src/reference/acquisition/camera.md

```diff
@@ -69,7 +67,7 @@ Events and commands from the camera are collected from and published to `Subject
 Here you set the names used for these `Subjects` to identify events and triggers for this specific camera.
 Each of these `Subjects` is published and becomes accessible in the Bonsai editor's toolbox anywhere in the workflow using its name.
 
-###### Device event subjects <!-- This was missing - is this correct? -->
+###### Device input subjects
```

Comment:
>Yes thank you! However, the device event subjects are subjects that are the result of the node. The Device command subjects are inputs. They inherited the 'events' and 'commands' names from harp, but could perhaps be changed to 'device input subjects' and 'device output subjects', or even dropping the 'device'.

Thanks for the clarification @NeuroThom. Having second thoughts about this change here, as everywhere else we use "Device command subjects", unless we want to change all of these, i.e. Device command subjects &rarr; Device input subjects; Device event subjects &rarr; Device output subjects - we also have "Oher output subjects" ?